packages/core/src/Editor.ts

```diff
@@ -80,6 +81,7 @@ export class Editor extends EventEmitter<EditorEvents> {
     enablePasteRules: true,
     enableCoreExtensions: true,
     enableContentCheck: false,
+    addTypeAttributes: true,
```

Comment:
I have the feeling that this is not really clear. But I don't have any alternatives either 😂

`enableHelperAttributes`, `renderDOMHelperAttributes`, …

Reply:
Removed it in favor of just using the `enableCoreExtensions` options. So by default it's disabled via `enableCoreExtensions: { typenames: false }` - but can be enabled by just setting this to true.

This is how we do it for the other core extensions already.